### PR TITLE
Add a section to configuration docs for Unified Alerting screenshots.

### PR DESCRIPTION
**Pull request by joeblubaugh**
As noted in #50350, we need a section in the configuration documents for our new parameters.

### CODE REVIEW

File: docs/sources/setup-grafana/configure-grafana/_index.md [hunk 1]
```diff
@@ -1259,6 +1259,25 @@ The interval string is a possibly signed sequence of decimal numbers, followed b
 
 <hr>
 
+## [unified_alerting.screenshots]
+
+### capture
+
+Enable screenshots in notifications. This option requires a remote HTTP image rendering service. Please see [[rendering]]({{< relref "#rendering">}}) for further configuration options.
+
```
Comment (brendamuir) [inline docs/sources/setup-grafana/configure-grafana/_index.md:1267]:
```suggestion
Enable screenshots in notifications. This option requires a remote HTTP image rendering service. For more configuration options, refer to [[rendering]]({{< relref "#rendering">}}).

```

File: docs/sources/setup-grafana/configure-grafana/_index.md [hunk 2]
```diff
@@ -1259,6 +1259,25 @@ The interval string is a possibly signed sequence of decimal numbers, followed b
...
+### max_concurrent_screenshots
+
+The maximum number of screenshots that can be taken at the same time. This option is different from
+concurrent_render_request_limit as max_concurrent_screenshots sets the number of concurrent screenshots
+that can be taken at the same time for all firing alerts where as concurrent_render_request_limit sets
+the total number of concurrent screenshots across all Grafana services.
```
Comment (brendamuir) [inline docs/sources/setup-grafana/configure-grafana/_index.md:1273]:
```suggestion
The maximum number of screenshots that can be taken at the same time. This option is different from
concurrent_render_request_limit.  
* max_concurrent_screenshots sets the number of concurrent screenshots
that can be taken at the same time for all firing alerts.
* concurrent_render_request_limit sets the total number of concurrent screenshots across all Grafana services.
```

File: docs/sources/setup-grafana/configure-grafana/_index.md [hunk 3]
```diff
@@ -1259,6 +1259,25 @@ The interval string is a possibly signed sequence of decimal numbers, followed b
...
+
+### upload_external_image_storage
+
+Uploads screenshots to the local Grafana server or remote storage such as Azure, S3 and GCS. Please
+see [[external_image_storage]]({{< relref "#external_image_storage" >}}) for further configuration options. If this option is false then
+screenshots will be persisted to disk for up to temp_data_lifetime.
```
Comment (brendamuir) [inline docs/sources/setup-grafana/configure-grafana/_index.md:1279]:
```suggestion
Uploads screenshots to the local Grafana server or remote storage such as Azure, S3 and GCS. For more configuration options, refer to [[external_image_storage]]({{< relref "#external_image_storage" >}}). If this option is false,screenshots will be persisted to disk for up to temp_data_lifetime.
```

**Review by brendamuir — APPROVED** [review]

File: docs/sources/setup-grafana/configure-grafana/_index.md [hunk 4]
```diff
@@ -1259,6 +1259,24 @@ The interval string is a possibly signed sequence of decimal numbers, followed b
 
 <hr>
 
+## [unified_alerting.screenshots]
+
+### capture
+
+Enable screenshots in notifications. This option requires a remote HTTP image rendering service. For more configuration options, refer to [[rendering]]({{< relref "#rendering">}}).
```
Comment (gotjosh) [inline docs/sources/setup-grafana/configure-grafana/_index.md:1266]:
```suggestion
Enable screenshots in notifications. This option requires a remote HTTP image rendering service. For more configuration options, refer to [[rendering]]({{< relref "#rendering">}}). The default value is `false`.
```

File: docs/sources/setup-grafana/configure-grafana/_index.md [hunk 5]
```diff
@@ -1259,6 +1259,24 @@ The interval string is a possibly signed sequence of decimal numbers, followed b
...
+
+### max_concurrent_screenshots
+
+The maximum number of screenshots that can be taken at the same time. This option is different from
+concurrent_render_request_limit.  
+* max_concurrent_screenshots sets the number of concurrent screenshots
+that can be taken at the same time for all firing alerts.
+* concurrent_render_request_limit sets the total number of concurrent screenshots across all Grafana services.
```
Comment (gotjosh) [inline docs/sources/setup-grafana/configure-grafana/_index.md:1274]:
```suggestion
* concurrent_render_request_limit sets the total number of concurrent screenshots across all Grafana services.

The default value is `5`.
```

File: docs/sources/setup-grafana/configure-grafana/_index.md [hunk 6]
```diff
@@ -1259,6 +1259,24 @@ The interval string is a possibly signed sequence of decimal numbers, followed b
...
+
+### upload_external_image_storage
+
+Uploads screenshots to the local Grafana server or remote storage such as Azure, S3 and GCS. For more configuration options, refer to [[external_image_storage]]({{< relref "#external_image_storage" >}}). If this option is false,screenshots will be persisted to disk for up to temp_data_lifetime.
```
Comment (gotjosh) [inline docs/sources/setup-grafana/configure-grafana/_index.md:1278]:
```suggestion
Uploads screenshots to the local Grafana server or remote storage such as Azure, S3 and GCS. For more configuration options, refer to [[external_image_storage]]({{< relref "#external_image_storage" >}}). If this option is false,screenshots will be persisted to disk for up to temp_data_lifetime. The default value is `false`.
```

**Review by gotjosh — APPROVED** [review]
LGTM, but please see my comment so that it matches the others above.

**Comment by joeblubaugh** [comment]
A different change was merged that adds this documentation. https://github.com/grafana/grafana/commit/7e2870b7bb5c0f1be9f83429ac6c97ba896fa91d

**Pull request closed without merging**